contracts/scripts/deploy.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @notice Umbra Deployment script
+ * @dev At each contract deployment or transaction, we wait for the transaction to be mined before
+ * continuing to the next step. This is done because it makes it simpler to continue from that spot
+ * rather than restart the full deployment
+ * @dev To initialize a deploy:
+ *   `yarn deploy:umbra --network <network>`   (where network specifies a netwrk found in the hardhat.config.ts file)
+ * If deploying to a local node (--network localhost), first start Hardhat using `yarn hardhat node`
+ */
+const fs = require('fs');
+const hre = require('hardhat');
+const { ethers } = hre;
+
+const RINKEBY_PAYMASTER_PUBLIC_RELAYER_ADDRESS = '0x53C88539C65E0350408a2294C4A85eB3d8ce8789';
```

Comment:
I think we should move this into `deployParams.json`, then modify `deployParams.json` as explained in the comment at that file

Reply:
Done

Reply:
Agreed that it'd be nice to move this into the json config.

Reply:
Good idea.. moved